packages/core/src/external/carequality/saml/security/security-header.ts

```diff
@@ -0,0 +1,179 @@
+import * as pkijs from "pkijs";
+import * as asn1js from "asn1js";
+import { arrayBufferToString, toBase64 } from "pvutils";
+import { EnvType } from "../../../../util/env-var";
+import { SNOMED_CODE, NHIN_PURPOSE_CODE_SYSTEM } from "../../shared";
+import { namespaces } from "../namespaces";
+
+export const security_header_timestamp_id = "TS-7c229e85-d62b-471e-9112-a49d1c365004";
+export const security_header_enveloped_id = "TS_3e57269d-075d-4d3f-9f5d-c97ad6afc009";
+
+export function createSecurityHeader(
+  x509CertPem: string,
+  created_timestamp: string,
+  expires_timestamp: string,
+  to_url: string,
+  subject_role: string,
+  metriport_organization: string,
+  home_community_id: string,
+  purpose_of_use: string,
+  envType?: EnvType
```

Comment:
I see. I'd not add code that we're not using yet. We might change our minds before/during the next PR and this could end up lingering, and future selves would be like WTF is this for? 😄